acceptance-tests/build.gradle

```diff
@@ -120,6 +120,7 @@ task acceptanceTest(dependsOn: [
   description = 'Runs Web3Signer acceptance tests.'
   group = 'verification'
 
+  //systemProperty 'debugSubProcess', 'true'
```

Comment:
If enabled it will allow you to debug ATs locally since for those web3signer is run on a separate process.
I thought leaving it in was better than it being hidden in the code.
Another option would be to uncomment and set to false.
Or could document it in the README maybe

Reply:
think leaving it is fine